target/linux/ath79/image/Makefile

```diff
@@ -47,6 +47,12 @@ define Build/append-loader-okli-uimage
 	cat "$(KDIR)/loader-$(word 1,$(1)).uImage" >> "$@"
 endef
 
+define Build/prepend-loader-okli
+	cat "$(KDIR)/loader-$(word 1,$(1)).$(LOADER_TYPE)" > "$@.new"
+	cat "$@" >> "$@.new"
+	mv "$@.new" "$@"
+endef
+
```

Comment:
Short description of arguments / How to use

Reply:
Removed it in the end since it is equivalent to loader-okli with bs=1